### ISSUE TITLE
Elements in BPMN and DMN diagrams should be keyboard-accessible

### ISSUE THREAD

**Issue by lmbateman** [camunda/camunda-modeler#5689]
### Problem you would like to solve

The ability to navigate through a diagram using the keyboard has been requested. This would help us satisfy [WCAG SC 2.1.1](https://www.w3.org/TR/WCAG22/#keyboard), which is a Level A requirement:

> Success Criterion 2.1.1 Keyboard (Level A)
> All [functionality](https://www.w3.org/TR/WCAG22/#dfn-functionality) of the content is operable through a [keyboard interface](https://www.w3.org/TR/WCAG22/#dfn-keyboard-interface) without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

I am submitting this as a feature request rather than a bug because we may be exempt from the requirement based on the last clause.

### Proposed solution

I should be able to navigate through a diagram using only my keyboard. Some example solutions:
1. For gateways, provide a popup with numbers listing the possible elements I can navigate to. I can then use the number keys to move to the desired next element in the flow.
2. We could use the same mechanism for ad hoc subprocesses.
3. Boundary events could be just the next item in the focus sequence.
4. Etc....

### Alternatives considered

Leave things as they are.

**Comment by barmac** [comment]
Backlog as it needs dedicated effort. We need to figure out how this should work.

**Comment by alibama** [comment]
great question - has this gone anywhere?  seems like a useful way to help organizations that are remediating content for part of title II ada compliance.... i know it's niche, but still it would help check an important box for some folks
